src/elements-format.ts

```diff
@@ -15,9 +15,9 @@
 /**
  * The base interface, holding properties common to many nodes.
  */
-export interface Node {
+export interface Feature {
```

Comment:
`Node` was too general, and Feature is how the rest of the code refers to this kind of thing